three/three.py

```diff
@@ -72,6 +72,12 @@ def decimal_places(i):
 def days_ago(timestamp):
     return timestamp - (86400 * 3)
 
+def third_element(items):
+	try:
+		return items[2]
```

Comment:
```suggestion
		return items[three()-1]
```

Reply:
I like the suggestion from @buckley-w-david. 
@suparna13 please add and verify a test case and I will merge.

Reply:
See the new [contributing guide](https://github.com/JasonTurley/three/blob/master/CONTRIBUTING.md) if unsure on any steps.

Reply:
@JasonTurley Added requested changes and test case as well.